messages/warning-pinnacle.rst

```diff
@@ -0,0 +1,21 @@
+.. WARNING::
```

Comment:
Was this extra file in the root directory intended?

Reply:
No that was not.

Reply:
> Was this extra file in the root directory intended?

No it was not. thank you for catching and fixing that